rotkehlchen/tests/unit/decoders/test_1inch.py

```diff
@@ -934,3 +935,53 @@ def test_1inchv4_swap_on_polygon(polygon_pos_inquirer, polygon_pos_accounts):
         ),
     ]
     assert expected_events == events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x4df4Ab179e3FdaC1930cd0a610D6bA6D5808E8Ff']])
+def test_1inch4_swap_via_defi_plaza(ethereum_inquirer, ethereum_accounts):
+    tx_hash = deserialize_evm_tx_hash('0xa99ce12e628620861215c88ee2c51f6c0468442dfe0504c9e5f2c918cf63fc8c')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        tx_hash=tx_hash,
+    )
+    timestamp, user, gas, amount_in, amount_out = TimestampMS(1657035153000), ethereum_accounts[0], '0.01345755735660528', '790.785045156438592337', '710.921537'  # noqa: E501
+    assert events == [
+        EvmEvent(
```

Comment:
```suggestion
    assert events == [EvmEvent(
```
can save a couple lines and de-indent the event list.